extensions/hibernate-search-orm-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/orm/elasticsearch/runtime/devconsole/HibernateSearchSupplier.java

```diff
@@ -21,24 +24,34 @@ public class HibernateSearchSupplier implements Supplier<List<DevUiIndexedEntity
     }
 
     @Override
-    public List<DevUiIndexedEntity> get() {
+    public Map<String, List<DevUiIndexedEntity>> get() {
```

Comment:
While `Map<String, List<...>>` definitely works, I think you'll be better off with proper "data" classes for the persistence units, similar to what was done for the Hibernate ORM extension; see `io.quarkus.hibernate.orm.runtime.devconsole.HibernateOrmDevConsoleInfoSupplier.PersistenceUnitsInfo` and `io.quarkus.hibernate.orm.runtime.devconsole.HibernateOrmDevConsoleInfoSupplier.PersistenceUnitInfo`.

Reply:
done